Content.Trauma.Client/Botany/PlantAnalyzer/UI/PlantAnalyzerBoundUserInterface.cs

```diff
@@ -71,18 +71,18 @@ protected override void Dispose(bool disposing)
     }
     public void DeleteDatabaseEntry(int index)
     {
-        if (index < 0)
+        if (index < -1)
```

Comment:
this will still allow a index of -1 to be sent

the validation should also be done on server